scripts/dandiset_000004_converted.yaml

```diff
@@ -0,0 +1,240 @@
+about:
+- identifier: MTL
+  name: Medial Temporal Lobe
+access:
+- email: nand.chandravadia@cshs.org
+  status: Open
+contributor:
+- affiliations:
+  - Department of Neurosurgery, Cedars-Sinai Medical Center, Los Angeles, CA, USA
+  email: nand.chandravadia@cshs.org
+  identifier:
+    propertyID: ORCID
+    value: 0000-0003-0161-4007
```

Comment:
This doesn't match the schema in `models.py`. According to that definition, the `Person` class should have properties of the following form:

```
...
identifier: {
  identifier: {
    ...,
    propertyID: ...,
    value: ...
  }
}
...
```

Personal input, what you have here seems more sane. I don't see the need for the nested `identifier` field. It seems to me the `Identifier` type could just represent `Union[str, AnyUrl, PropertyValue]`.